stdlib/java/src/main/ballerina/src/java/ballerina_values_to_Java.bal

```diff
@@ -15,46 +15,62 @@
+# + value - The `handle` whose referred value to be converted to Ballerina `string`
+# + return - The Ballerina `string` representation of the Java object referred by the `handle` or
 #            returns `nil` of the `handle` refers to Java null
 public function toString(handle value) returns string? = external;
 
 # Returns `true` if this handle refers to Java null.
+# ```ballerina
+# boolean status = java:isNull(value);
+# ```
 #
-# + value - the `handle` whose referred value to be tested with Java null
+# + value - The `handle` whose referred value to be tested with Java null
 # + return - `true` if this handle refers to Java null
 public function isNull(handle value) returns boolean = external;
 
 # Returns a `handle` that refers to Java null.
+# ```ballerina
+# handle nullHandle = java:createNull();
+# ```
 #
-# + return - the `handle` that refers to Java null
+# + return - The `handle` that refers to Java null
 public function createNull() returns handle = external;
 
 # Returns a `handle` that refers to the Java Class object associated
 # with the class or interface with the given string name.
+# ```ballerina
+# handle|error intClass = java:getClass("int");
+# ```
 #
 # This function performs a Java `class.forName(name)` except for following cases:
 #
-#   name        output
-#   boolean     the Java Class instance representing the Java primitive type boolean
-#   byte        the Java Class instance representing the Java primitive type byte
-#   char        the Java Class instance representing the Java primitive type char
-#   short       the Java Class instance representing the Java primitive type short
-#   int         the Java Class instance representing the Java primitive type int
-#   long        the Java Class instance representing the Java primitive type long
-#   float       the Java Class instance representing the Java primitive type float
-#   double      the Java Class instance representing the Java primitive type double
-# + name - the name of the Java class
-# + return - the Java Class object for the class with the given name.
+# | Name     |   Output                                                           |
+# |:--------:|:------------------------------------------------------------------:|
+# | boolean  |   Java Class instance representing the Java primitive type boolean |
+# | byte     |   Java Class instance representing the Java primitive type byte    |
+# | char     |   Java Class instance representing the Java primitive type char    |
+# | short    |   Java Class instance representing the Java primitive type short   |
+# | int      |   Java Class instance representing the Java primitive type int     |
+# | long     |   Java Class instance representing the Java primitive type long    |
+# | float    |   Java Class instance representing the Java primitive type float   |
+# | double   |   Java Class instance representing the Java primitive type double  |
```

Comment:
```suggestion
# | double   |   Java Class instance representing the Java primitive type double  |
```
Build locally and make sure the table format gets rendered and displayed properly.

Reply:
The page which directs when you click on the function shows it properly (**module/function.html**)

![Selection_126](https://user-images.githubusercontent.com/8381003/79036131-bc49cd00-7be2-11ea-88a0-5ee88aa70af0.png)

But the module page, which shows module.md and other functions altogether does not seem to indent properly (**module/index.html**)
![Selection_127](https://user-images.githubusercontent.com/8381003/79036140-d97e9b80-7be2-11ea-94ad-d1e44653b790.png)

Reply:
Please create an issue to report the rendering issue.